### DISCUSSION TITLE
Untiy 3.2.0 support

### DISCUSSION THREAD

**Discussion by mostafa-abdelbrr** [AssetRipper/AssetRipper#939] [Q&A] [answered]
Any plans to support Unity 3.2.0? I know the textures and stuff get extracted, but scripts don't, even if I drag and drop the dlls.

**Comment by ds5678** [comment, marked as answer]
No, there is not, for multiple reasons:
* I do not have type trees from before `3.4.0`.
* Supporting anything before `3.5.0` adds a lot of complexity to AssetRipper.

**Reply by mostafa-abdelbrr** [reply]
And as such I shouldn't submit any error reports for problems when extracting scripts from such a version, right?

**Reply by ds5678** [reply]
Correct.

You might check out uTinyRipper or early versions (before 0.2.0.0) of AssetRipper. They had better support for old Unity versions.

**Reply by mostafa-abdelbrr** [reply]
Thank you for the info! I will be sure to check them out!
